server/command/auto_respond.go

```diff
@@ -3,19 +3,39 @@ package command
 import (
 	"fmt"
 	"strings"
-)
 
-const autoRespondHelp = "### Autorespond commands:\n" +
-	"`/mscalendar autorespond <message>` - Set the autorespond message\n"
+	"github.com/mattermost/mattermost-server/v5/model"
 
-func (c *Command) autoRespond(parameters ...string) (string, bool, error) {
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/config"
+)
 
+func (c *Command) autoRespond(parameters ...string) (string, bool, error) {
 	if len(parameters) == 0 {
-		return autoRespondHelp, false, nil
+		request := model.OpenDialogRequest{
+			TriggerId: c.Args.TriggerId,
+			URL:       c.Config.PluginURL + config.PathDialog + config.PathAutoRespond,
+			Dialog: model.Dialog{
+				CallbackId: "",
+				Title:      "Auto Respond Message",
+				Elements: []model.DialogElement{
+					{
+						DisplayName: "Autorespond Message",
+						Name:        "auto_respond",
```

Comment:
I agree with the suggested change for the dialog field above. Regarding variable names, I think they are already consistent. They're all `autoRespond`, treating it as two words.

As far as text being displayed to the user, I guess the reason I felt I should go with "autorespond message" is that "auto respond message" feels ambiguous between (auto respond) message and auto (respond message), but I think the grammatically correct thing to do in such cases is hyphenat.

So, for the Welcome/Setting titles or places where it's just "auto respond", it can be two separate words, but when it's "auto-respond message", it can be hyphenated. Does that sound right?

Reply:
Sounds good to me :+1:

Reply:
OK, so it will have "Auto Respond" as a title, "Auto-Respond Message" as a title (according to https://blog.apastyle.org/apastyle/2012/03/title-case-and-sentence-case-capitalization-in-apa-style.html), "auto-respond message" in a sentence, "Auto-respond message" at the beginning of a sentence (https://ontariotraining.net/writing-style-capitalization-of-hyphenated-words%E2%80%A8%E2%80%A8/).